lib/engines/content_block_manager/lib/tasks/update_rates_cadence_to_frequency.rake

```diff
@@ -4,11 +4,11 @@ namespace :content_block_manager do
     ContentBlockManager::ContentBlock::Document.where(block_type: "pension").find_each do |document|
       document.editions.each do |edition|
         edition.details["rates"]&.each do |key, value|
-          if value["cadence"]
-            puts "updating #{key} on #{edition.title}"
-            edition.details["rates"][key]["frequency"] = value["cadence"]
-            edition.details["rates"][key].delete("cadence")
-          end
+          next if value["cadence"].blank?
+
+          puts "updating #{key} on #{edition.title}"
+          edition.details["rates"][key]["frequency"] = value["cadence"]
+          edition.details["rates"][key].delete("cadence")
```

Comment:
Does this belong in the previous commit?

Reply:
ah yep good catch!